generic_expression_patterns_modules/other_enrichment_methods.R

```diff
@@ -0,0 +1,161 @@
+# Alternative enrichment analyses to GSEA
+# These are methods were implemented, expecting
+# RNA-seq input. These will need to be adjusted if
+# using microarray data
+  
+# Load libraries
+library("fgsea")
+library("limma")
+library("GSVA")
+library("DEFormats")
+library("edgeR")
+library("DESeq2")
+library("clusterProfiler")
+
+find_enriched_pathways_ROAST <- function(expression_filename,
```

Comment:
Maybe it would be helpful to comment inside of each of these functions with a citation to the paper where each method is proposed or described (or refer people to the notebook `2_apply_enrichment_method.ipynb` where you have a nice description of each of them)?